setup.py

```diff
@@ -34,7 +34,7 @@
     keywords=["gis"],
     install_requires=[
         "agrc-sweeper==1.3.*",
```

Comment:
**question**: didn't we decide to update python version comparisons to another format to avoid the weird dependabot pull requests that were generated from this?

Reply:
I would love to know how to make dependabot python PRs more relevant! I didn't know that it was related to the `*`. What's the better format?